src/updateVotes.js

```diff
@@ -0,0 +1,39 @@
+import * as api from '../src/api';
+const updateVotes = (setShowError, setItems, id, itemType) => {
```

Comment:
Could be more descriptive instead of simply saying 'items'.

Reply:
Thanks very much for the feedback :)